loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -423,67 +465,105 @@ static void binary_update_same_version_test(void)
 static void binary_update_new_version_test(void)
 {
 	int ret;
-	char filepath[CONFIG_PATH_MAX];
+	char filepath[BINARY_PATH_LEN];
 	binary_update_info_t pre_bin_info;
 	binary_update_info_t cur_bin_info;
 
-	binary_update_getinfo(APP_NAME, &pre_bin_info);
+	ret = binary_update_getinfo(APP_NAME, &pre_bin_info);
+	if (ret != OK) {
+		return;
+	}
 
 	/* Copy current binary and update version. */
-	binary_update_download_binary(&pre_bin_info, DOWNLOAD_VALID_BIN);
+	ret = binary_update_download_binary(&pre_bin_info, DOWNLOAD_VALID_BIN);
+	if (ret != OK) {
+		return;
+	}
 
-	binary_update_reload(APP_NAME);
+	ret = binary_update_reload(APP_NAME);
+	if (ret != OK) {
+		return;
+	}
 
 	sleep(2);
 
-	binary_update_getinfo(APP_NAME, &cur_bin_info);
+	ret = binary_update_getinfo(APP_NAME, &cur_bin_info);
+	if (ret != OK) {
+		return;
```

Comment:
yes... like this funcion. Keep void is very dangerous but same as above. you can ignore because it just sample app.

Reply:
Each functions which have a name binary_update_XXX_test are independent so it doesn't have to return result I think.